app/src/main/java/com/arygm/quickfix/ui/home/MessagesScreen.kt

```diff
@@ -1,146 +1,396 @@
+                                            color = MaterialTheme.colorScheme.onBackground,
+                                            textAlign = TextAlign.Center,
+                                            modifier = Modifier.padding(bottom = maxHeight * 0.02f))
+                                        Row(
+                                            horizontalArrangement =
+                                                Arrangement.spacedBy(maxWidth * 0.1f),
+                                            modifier = Modifier.fillMaxWidth(0.5f)) {
+                                              // Accept button
+                                              IconButton(
+                                                  onClick = {
+                                                    coroutineScope.launch {
+                                                      Log.e("hhaha", " acceptite")
+                                                      chat.let {
+                                                        val updatedChat =
+                                                            it.copy(
+                                                                chatStatus =
+                                                                    ChatStatus.GETTING_SUGGESTIONS)
+                                                        chatViewModel.updateChat(
+                                                            updatedChat, {}, {})
+                                                      }
+                                                    }
+                                                  },
+                                                  modifier =
+                                                      Modifier.weight(0.1f)
+                                                          .aspectRatio(1f)
+                                                          .background(
+                                                              MaterialTheme.colorScheme.surface,
+                                                              CircleShape)
+                                                          .testTag("acceptButton")) {
+                                                    Icon(
+                                                        imageVector = Icons.Default.Check,
+                                                        contentDescription = "Accept",
+                                                        tint = colorScheme.primary,
+                                                        modifier = Modifier.fillMaxSize(0.8f))
+                                                  }
+                                              // Reject button
+                                              IconButton(
+                                                  onClick = {
+                                                    coroutineScope.launch {
+                                                      Log.e("hhaha", "9alwa")
```

Comment:
Maybe remove those comments 🤣

Reply:
Thanks! I was using it for debugging and forgot to remove it.